assets/cms/components/gallery/GalleryEdit.vue

```diff
@@ -0,0 +1,161 @@
+<template>
+    <div class="ccm-gallery-edit">
+
+        <ul id="galleryBlock" class="nav nav-tabs" role="tablist">
+            <li class="nav-item">
+                <a class="nav-link"
+                    :class="activeTab === 'image' ? 'active' : ''"
+                    @click="openTab('image')">
+                    Images
+                </a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link"
+                    @click="openTab('design')"
+                    :class="activeTab === 'design' ? 'active' : ''">
+                    Design
+                </a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link"
+                    @click="openTab('settings')"
+                    :class="activeTab === 'settings' ? 'active' : ''">
+                    Settings
+                </a>
+            </li>
+        </ul>
+
+        <div class="tab-content" id="galleryBlockContent">
+            <div v-if="activeTab === 'image'"  id="galleryImages">
+                <div class="text-right mt-4">
+                    <button class="btn btn-secondary" @click="addImage()">Add Images</button>
+                </div>
+                <div class="image-container mt-4"
+                    ref="imageContainer"
+                    :class="Object.keys(this.activeImage).length != 0 ? 'active-image':''">
+                    <ImageCell v-for="(image, index) in $props.gallery" :key="index"
+                        :src="image.thumbUrl"
+                        :file-size="image.fileSize"
+                        size="120"
+                        :isActive="activeImage.id === image.id ? true : false"
+                        @click="openImage(image, $event)"
+                        @delete="deleteImage(image)"
+                        />
+                </div>
+
+                <div v-if="Object.keys(activeImage).length != 0">
```

Comment:
Using `null` for the default / closed value instead of `{}` would make this easier to check without having to do this:
`v-if='activeImage'`

Reply:
Totally agree, I just updated this as well.